src/base-input.js

```diff
@@ -767,14 +767,12 @@ export default class BaseInput extends LitElement {
    * @returns {void}
    */
   configureDataForType() {
-    if (this.type === 'month-day-year') {
+    if (this.type === 'month-day-year' || 'year-month-day') {
```

Comment:
These updates broke tests 
https://github.com/AlaskaAirlines/auro-input/actions/runs/4640862496/jobs/8213233895#step:5:95

This should have been picked up in the pre-commit?